client/src/domain/home/Home.tsx

```diff
@@ -1,12 +1,89 @@
 import React from "react";
+import { useTranslation } from "react-i18next";
 
+import CollectionCardContainer from "../../common/components/collectionCard/CollectionCardContainer";
+import Hero from "../../common/components/hero/Hero";
+import Container from "../app/layout/Container";
 import Layout from "../app/layout/Layout";
+import styles from "./home.module.scss";
 import Search from "./Search";
 
 const Home: React.FC = () => {
+  const { t } = useTranslation();
   return (
     <Layout>
-      <Search />
+      <Hero
+        buttonText={t("home.hero.buttonText")}
+        subTitle={t("home.hero.subTitle")}
+        title={t("home.hero.title")}
+      />
+      <Container>
+        <Search />
+      </Container>
+      <div className={styles.collectionCardContainer}>
+        <Container>
+          <h2>{t("home.collections.title")}</h2>
+          <CollectionCardContainer
+            cards={[
+              {
+                count: 140,
```

Comment:
This looks like hardcoded stuff, a TODO referencing a Jira issue could be helpful in keeping track of this

Reply:
Good point. Added a comment